src/credentials/CHIPCert.h

```diff
@@ -436,7 +441,8 @@ class DLL_EXPORT ChipCertificateSet
      *
      * @return Returns a CHIP_ERROR on error, CHIP_NO_ERROR otherwise
      **/
-    CHIP_ERROR LoadCert(chip::TLV::TLVReader & reader, BitFlags<CertDecodeFlags> decodeFlags);
+    CHIP_ERROR LoadCert(chip::TLV::TLVReader & reader, BitFlags<CertDecodeFlags> decodeFlags, const uint8_t * chipCert = nullptr,
```

Comment:
Maybe the method should return pointer to the ChipCertificateData? This way the caller will be able to extract all needed certificate fields from ChipCertificateData struct.